src/signature/utils.js

```diff
@@ -0,0 +1,11 @@
+module.exports.hasConfig = function(config, endpoint) {
```

Comment:
This method is returning the field level config, not just checking it's presence. So please name it appropriately:
```suggestion
module.exports.getFieldLevelConfig = function(config, endpoint) {
```